docs/security/access-control.md

```diff
@@ -152,9 +152,9 @@ You can also create a dedicated role for your business by API or the client.
 
 ### Permission privileges
 
-| Name          | Supports Securable Object | Operation              |
-|---------------|---------------------------|------------------------|
-| MANAGE_GRANTS | Metalake                  | grant or revoke a role |
+| Name          | Supports Securable Object | Operation                             |
+|---------------|---------------------------|---------------------------------------|
+| MANAGE_GRANTS | Metalake                  | grant or revoke a role or a privilege |
```

Comment:
grant or revoke a role?  It seems a bit strange. 

should it be `grant or revoke a privilege for a role` ?

Reply:
Grant/Revoke a role for groups/users. Grant Revoke a privilege for roles.

Reply:
ok

Reply:
I modified here.